sourcegraph/jetbrains
# Replace CHANGELOG.md with hardcoded link to GitHub releases page

Fixes: https://github.com/sourcegraph/sourcegraph/issues/57890

### Test plan

* Install plugin and open "What's New"
* Verify: description links to GitHub

### Demo

![image](https://github.com/sourcegraph/jetbrains/assets/5013708/92cbe70c-79b7-4465-a725-ae8fb7166fec)
